src/System.Security.Cryptography.Pkcs/src/Resources/Strings.resx

```diff
@@ -100,4 +100,7 @@
   <data name="InvalidOperation_WrongOidInAsnCollection" xml:space="preserve">
     <value>AsnEncodedData element in the collection has wrong Oid value: expected = '{0}', actual = '{1}'.</value>
   </data>
+  <data name="PlatformNotSupported_CryptographyPkcs" xml:space="preserve">
+    <value>System.Security.Cryptography.Pkcs is not yet supported on this platform</value>
```

Comment:
@danmosemsft By the by, I was just pointing out why this one was different when I sent it to Sepideh.  If anything else is also point-in-time, I'm happy to conform.  Or if we just say "well, the normal PNSE message wouldn't say 'yet'", I'll accept that, too.